[2025-03-17T20:20:14.122+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T20:20:14.931+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.user_sessions_project_c_db scheduled__2025-03-17T20:10:00+00:00 [queued]>
[2025-03-17T20:20:15.272+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.user_sessions_project_c_db scheduled__2025-03-17T20:10:00+00:00 [queued]>
[2025-03-17T20:20:15.292+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T20:20:15.522+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.user_sessions_project_c_db> on 2025-03-17 20:10:00+00:00
[2025-03-17T20:20:15.813+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3537) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T20:20:15.852+0000] {standard_task_runner.py:72} INFO - Started process 3557 to run task
[2025-03-17T20:20:15.852+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.user_sessions_project_c_db', 'scheduled__2025-03-17T20:10:00+00:00', '--job-id', '251', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp2yv4zvc2']
[2025-03-17T20:20:15.931+0000] {standard_task_runner.py:105} INFO - Job 251: Subtask load_tables.user_sessions_project_c_db
[2025-03-17T20:20:18.161+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.user_sessions_project_c_db scheduled__2025-03-17T20:10:00+00:00 [running]> on host c032966e105e
[2025-03-17T20:20:20.860+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.user_sessions_project_c_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T20:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-17T20:10:00+00:00'
[2025-03-17T20:20:20.871+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T20:20:21.840+0000] {sql.py:278} INFO - Executing: select load_tables('user_sessions_project_c_db', 'user_sessions');
[2025-03-17T20:20:22.290+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T20:20:22.540+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T20:20:22.630+0000] {sql.py:553} INFO - Running statement: select load_tables('user_sessions_project_c_db', 'user_sessions');, parameters: None
[2025-03-17T20:20:22.810+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T20:20:23.570+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T20:20:23.579+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.user_sessions_project_c_db, run_id=scheduled__2025-03-17T20:10:00+00:00, execution_date=20250317T201000, start_date=20250317T202014, end_date=20250317T202023
[2025-03-17T20:20:24.491+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T20:20:24.821+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
